src/slskd/Options.cs

```diff
@@ -837,7 +833,7 @@ public class IntegrationOptions
             ///     Gets FTP options.
             /// </summary>
             [Validate]
-            public FTPOptions FTP { get; private set; } = new FTPOptions();
+            public FtpOptions Ftp { get; private set; } = new FtpOptions();
```

Comment:
The casing changed here because the underscore strategy for the yaml library in use would serialize this as `f_t_p`.  Pascal case is in alignment with .NET standard practices, also.